controllers/metallb_controller.go

```diff
@@ -46,7 +46,20 @@ type MetallbReconciler struct {
 	Scheme *runtime.Scheme
 }
 
-var ManifestPath = "./bindata/deployment"
+const (
+	MetalLBDeployment    = "./bindata/deployment"
+	MetalLBOCPDeployment = "./bindata/ocpdeployment"
+)
+
+var ManifestPath string
+
+func init() {
+	if len(os.Getenv("OCP_PLATFORM")) == 0 {
```

Comment:
I'd have this as more generic `DEPLOYMENT_PATH`

Reply:
no longer using envvar we detect the platform name runtime